internal/builder/known_builder.go

```diff
@@ -1,45 +1,61 @@
 package builder
 
-type SuggestedBuilder struct {
+type KnownBuilder struct {
 	Vendor             string
 	Image              string
 	DefaultDescription string
+	Suggested          bool
+	Trusted            bool
 }
 
-var SuggestedBuilders = []SuggestedBuilder{
+var KnownBuilders = []KnownBuilder{
 	{
 		Vendor:             "Google",
 		Image:              "gcr.io/buildpacks/builder:v1",
 		DefaultDescription: "GCP Builder for all runtimes",
+		Suggested:          true,
+		Trusted:            true,
 	},
 	{
 		Vendor:             "Heroku",
 		Image:              "heroku/builder:22",
 		DefaultDescription: "Heroku-22 (Ubuntu 22.04) base image with buildpacks for Go, Java, Node.js, PHP, Python, Ruby & Scala",
+		Suggested:          true,
+		Trusted:            true,
 	},
 	{
 		Vendor:             "Heroku",
 		Image:              "heroku/builder:20",
 		DefaultDescription: "Heroku-20 (Ubuntu 20.04) base image with buildpacks for Go, Java, Node.js, PHP, Python, Ruby & Scala",
+		Suggested:          true,
```

Comment:
According to the Heroku builder [README](https://github.com/heroku/cnb-builder-images/blob/660742804f7f40097913445ef8c8888d4a700da3/README.md?plain=1#L29C5-L29C22), we have `heroku/builder:20` marked as "Available", where `heroku/bulider:22` is marked as "Recommended". With that in mind and `heroku-24` on the horizon, I think it would make sense to turn off  recommended for `heroku/builder:20`.

Reply:
`heroku/builder:20` is marked as `Suggested: false` now